resume/routes.py

```diff
@@ -370,44 +370,29 @@ def send_reset_pass(user):
 @app.route("/reset_account",methods=["GET","POST"])
 def request_reset():
     if current_user.is_authenticated:
-        return redirect(url_for('hello')),200
+        return redirect(url_for('hello'),code=301)
     form = requestresetform()
     if form.validate_on_submit():
         req_user = user.query.filter_by(email=form.email.data).first()
         send_reset_pass(req_user)
         flash("Email has been sent with instructions to Reset Password")
-        return redirect(url_for("hello")),200
+        return redirect(url_for("hello"),code=301)
 
     return render_template("reset.html",title="Reset Request Form",form=form)
 
 @app.route("/reset_account/<token>",methods=["GET","POST"])
 def token_reset(token):
     if current_user.is_authenticated:
-        return redirect(url_for('hello')),200
+        return redirect(url_for('hello'),code=301)
     user_req = user.verify_token(token)
     if user_req is None:
         flash("Token is Invalid or Expired","warning")
-        return redirect(url_for('request_reset')),400
+        return redirect(url_for('request_reset'),code=301)
```

Comment:
user authenticated and token invalid should have different status codes

Reply:
can we have different status codes for user authenticated , redirect pages and errors